frontend/public/components/deploy-image.tsx

```diff
@@ -90,6 +92,10 @@ export class DeployImage extends React.Component<DeployImageProps, DeployImageSt
     this.setState({name: event.currentTarget.value});
   };
 
+  onLabelsChange: React.ReactEventHandler<HTMLInputElement> = labels => {
```

Comment:
This looks like the wrong type if the method doesn't take an event.

```suggestion
  onLabelsChange = (labels: string[]) => {
```